vscode-extension/src/util.ts

```diff
@@ -81,6 +81,12 @@ export function updateWebviewEditorThemeMode(webview: vscode.Webview) {
   });
 }
 
+export function getConfigurationTarget() {
```

Comment:
A potential concern I have is that while this is fine for writing the aiconfig values, for reading now this could get out of sync:

1. User has no defined Workspace, saves AIConfig settings in Global
2. User sets their workspace
3. User reads values from `vscode.ConfigurationTarget.Workspace` but they aren't defined in there

I personally do not feel this is a big deal for python interpreter. If there's none defined, we simply ask them to re-select one and define it in that workspace, which is annoying for user to have to re-select when they feel they're already done that, but not the worse and definitely sort of an edge case. 

Are there cases Sarmad where you can see this being an issue for the AIConfig Model Registry commands? If so, I'll let you handle the best solution for it since you're most familiar with that flow